packages/playwright/src/reporter.ts

```diff
@@ -143,14 +146,16 @@ export default class ReplayPlaywrightReporter implements Reporter {
   }
 
   getFixtureData(test: TestExecutionIdData) {
-    const id = this._getTestExecutionId(test);
-    this.fixtureData[id] ??= {
-      steps: [],
-      stacks: {},
-      filenames: new Set(),
-    };
-
-    return this.fixtureData[id];
+    return withSentrySync("GetFixtureData", () => {
```

Comment:
it could be nice to have some `withSentryTracking` that could wrap a whole class automatically